tempto-core/src/main/java/com/teradata/tempto/internal/query/QueryResultValueComparator.java

```diff
@@ -39,7 +39,7 @@
         implements Comparator<Object>
 {
 
-    private static final double DOUBLE_FUZZY_MATCH_THRESHOLD = 0.00000000001;
+    private static final double DOUBLE_FUZZY_MATCH_THRESHOLD = Math.pow(2,-53); 
```

Comment:
Rename this constant to something more descriptive, e.g. `MACHINE_EPSILON_DOUBLE_64B`.